src/Controls/src/Core/Handlers/Items/Android/Adapters/StructuredItemsViewAdapter.cs

```diff
@@ -139,7 +139,10 @@ protected RecyclerView.ViewHolder CreateHeaderFooterViewHolder(object content, D
 				var viewHolder = SimpleViewHolder.FromFormsView(formsView, context, ItemsView);
 
 				// Propagate the binding context, visual, etc. from the ItemsView to the header/footer
-				ItemsView.AddLogicalChild(viewHolder.View);
+				if (!ItemsView.ContainsLogicalChild(viewHolder.View))
```

Comment:
If you don't want to run the contains you could also check the parent on the view. If it equals `ItemsView` then it contains it.